internal/config/cache/cache.go

```diff
@@ -35,13 +35,13 @@ import (
 
 // Cache related keys
 const (
-	Enable     = "enable"
-	Endpoint   = "endpoint"
-	ObjectSize = "object_size"
+	Enable    = "enable"
+	Endpoint  = "endpoint"
+	BlockSize = "block_size"
 
-	EnvEnable     = "MINIO_CACHE_ENABLE"
-	EnvEndpoint   = "MINIO_CACHE_ENDPOINT"
-	EnvObjectSize = "MINIO_CACHE_OBJECT_SIZE"
+	EnvEnable    = "MINIO_CACHE_ENABLE"
+	EnvEndpoint  = "MINIO_CACHE_ENDPOINT"
+	EnvBlockSize = "MINIO_CACHE_BLOCK_SIZE"
```

Comment:
Did something change in mincache too ? I mean minimum object size is clear, minimum block size doesn't seem to have clear meaning at all

Reply:
We cache ranges as well so it's not object size @vadmeste

Reply:
oh maybe DataSize name is better?